ADA/databricks_notebooks.qmd

```diff
@@ -11,15 +11,15 @@ title: "Databricks Notebooks"
 Notebooks are a special kind of script that Databricks supports. They consist of code blocks and markdown blocks which can contain formatted text, links and images. Due to the ability to combine markdown with code they are very well suited to creating and documenting data pipelines, such as creating a core dataset that underpins your other products. They are particularly powerful when parameterised and used in conjuction with [Workflows](ADA/Databricks_workflows.qmd).
```

Comment:
Line break to remove under notebooks
![image](https://github.com/user-attachments/assets/9e54aff4-7a91-4916-83b0-06fa0fa49829)

Reply:
Still need to fix this one 
![image](https://github.com/user-attachments/assets/b5c4b84e-6607-4d3b-aacb-3a493b525849)

Reply:
😞